Paws4Me/Paws4Me/Controllers/ViewController.swift

```diff
@@ -61,25 +65,23 @@ extension ViewController: UITableViewDelegate, UITableViewDataSource {
         cell.index = indexPath.row
         cell.pet = adoptablepet
         cell.setNeedsLayout()
+        let bgColorView = UIView()
+        bgColorView.backgroundColor = UIColor(red: 255/255.0, green: 222/255.0, blue: 193/255.0, alpha: 1)
+        cell.selectedBackgroundView = bgColorView
        return cell
    }
-    
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        
-        performSegue(withIdentifier: "showPetSingleDetails", sender: self)
+        performSegue(withIdentifier: "PetSingleDetailsViewController", sender: self)
     }
-    
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        
         if let destination = segue.destination as? PetSingleDetailsViewController {
             guard let rowIndex = petTable.indexPathForSelectedRow?.row else {return}
             guard let pageItem = adoptPetObject?.page?[rowIndex] else {return}
-            
             if let pagePetName = pageItem.name {
             destination.nameOfPet = pagePetName
             }
             if let pagePetAge = pageItem.age {
-            destination.imagOfPet = pagePetAge
+            destination.ageOfPet = pagePetAge
```

Comment:
`petAge` would be a better name 😄 see explanation on other PR 😄

Reply:
Thank you for the suggestion. 👍🏻